assignments/objects.js

```diff
@@ -19,26 +19,71 @@ const example = {
 
 // Write your intern objects here:
 
+const mitzi = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
+  "gender": "F"
+}
+
+const kennan = {
+  "id": 2,
+  "name": "Kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  "speak": function() {
+    return `Hello my name is ${this.name}!`;
+  }
+}
+
+const keven = {
+  "id": 3,
+  "name": "Keven",
+  "email": "kmummery2@wikimedia.org",
+  "gender": "M"
+}
+
+const gannie = {
+  "id": 4,
+  "name": "Gannie",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "M"
+}
+
+const antonietta = {
+  "id": 5,
+  "name": "Antonietta",
+  "email": "adaine5@samsung.com",
+  "gender": "F",
+  "multiplyNums": function(num1, num2) {
```

Comment:
just as a bit of a whats to come, ES6 cleans up this syntax a bit and allows you to define object methods as such
```suggestion
const obj = {
    id: 3,
    name: "Ryan",
    multiplyNums(n1,n2) {
          return n1 * n2
     }
}
```

It's not really that much different but you end up dropping the colon and function reserved work you will see this mostly in classes and then in React again

Reply:
Also, quotes can be used but if they are valid JS identifiers you can drop them on keys
[ ] can be used for the way you did or if its not a variable most people use the . 
[ ] can also be used when the key is a variable - which is useful for forms in React

also something for the future is destructing which is very useful i.e.,

```
const objA = {
   name: "Ryan",
    age: 37,
    location: "PA"
}
console.log(`My name is ${objA.name} and I am ${objA.age} years old and come from ${objA.location}`)
```
**//==> My name is Ryan and I am 37 years old and come from PA** is logged to the console.
Notice that repetition can get annoying so you can use de-structuring to clean it up...

```
const objA = {
   name: "Ryan",
    age: 37,
    location: "PA"
}
const { name, age, location } = objA
console.log(`My name is ${name} and I am ${age} years old and come from ${location}`)
```
**//==> My name is Ryan and I am 37 years old and come from PA** is logged to the console.

It becomes more useful later on when you are trying to be a little less terse.  I thought you would like to see some examples of advanced ideas that will be on the horizon.